core/src/test/scala/tamer/SinkSpec.scala

```diff
@@ -2,11 +2,10 @@ package tamer
 
 import io.confluent.kafka.schemaregistry.ParsedSchema
 import log.effect.zio.ZioLogWriter.log4sFromName
-import org.apache.kafka.clients.producer.{ProducerRecord, RecordMetadata}
-import org.apache.kafka.common.{Metric, MetricName, TopicPartition}
+import org.apache.kafka.clients.producer.ProducerRecord
+import utils.{FailingFakeProducer, FakeProducer}
```

Comment:
if we namespace these we want to use the fully qualified package name here, ie
```suggestion
import tamer.utils.{FailingFakeProducer, FakeProducer}
```
alternatively, we can leave both of the above in the main package

Reply:
utils => tamer.utils